README.md

```diff
@@ -199,10 +201,10 @@ The `Archive.OutputMode.COMPOSED` option is the default value for `outputMode`.
 You can only specify the `resolution` for composed archives using the `ArchiveProperties` builder. If you set the `resolution` property and also set the `outputMode` property to `Archive.OutputMode.INDIVIDUAL`, the method will throw an `InvalidArgumentException`.
 
 The accepted values for `resolution` are:
+
```

Comment:
was this extra white-space intended?

Reply:
It wasn't done intentionally.  Was likely my markdown linter.